solutions/swb-reference/integration-tests/support/utils/settings.ts

```diff
@@ -15,6 +15,11 @@ interface Setting {
   terminatedEnvId: string;
   rootUserNameParamStorePath: string;
   rootPasswordParamStorePath: string;
+  researcherUserNameParamStorePath: string;
+  researcherPasswordParamStorePath: string;
+  projectAdminUserNameParamStorePath: string;
+  projectAdminPasswordParamStorePath: string;
+  projectForRolesTesting: string;
```

Comment:
1) we already have projectId above--could that be repurposed for this as well?
2) do we want our researcher and our project admin to be in different projects so that we could test the boundary better?

Reply:
Unless things go very poorly, this is all temporary infrastructure that will go away once we can generate accounts at test-time. If things do go poorly and we can't generate at test time, you're right that we'll need two sets of PAs and Researchers that are members of different projects.